codeHF/dpl-config_run3.json

```diff
@@ -789,16 +789,17 @@
     "hf-candidate-selector-lc-to-k0s-p": {
         "ptCandMin": "0",
         "ptCandMax": "50",
-        "usePidBach": "true",
         "pPidThreshold": "1",
         "nSigmaTpcMaxLowP": "2",
-        "nSigmaTpcMaxHighP": "9999",
-        "nSigmaTofMaxLowP": "9999",
+        "nSigmaTpcMaxHighP": "0",
+        "nSigmaTpcCombinedMaxLowP": "2",
+        "nSigmaTpcCombinedMaxHighP": "9999",
+        "nSigmaTofMaxLowP": "0",
         "nSigmaTofMaxHighP": "3",
-        "requireTofLowP": "false",
-        "requireTofHighP": "true",
-        "probBayesMinLowP": "-1",
-        "probBayesMinHighP": "-1",
+        "nSigmaTofCombinedMaxLowP": "9999",
+        "nSigmaTofCombinedMaxHighP": "3",
+        "probBayesMinLowP": "0.8",
+        "probBayesMinHighP": "0.8",
```

Comment:
Can you please confirm that the configurables appear in the dumped json in this order?
(Because in the task, their order is different.)

Reply:
You were right. I copied from an old dumped json from before I changed the order in the task

Reply:
Thanks, but please don't force push changes when a PR has already been reviewed. I cannot see the changes made after the review.